apt/private/index.bzl

```diff
@@ -11,13 +11,13 @@ _DEB_IMPORT_TMPL = '''\
 filegroup(
     name = "data",
     visibility = ["//visibility:public"],
-    srcs = ["data.tar.xz"]
+    srcs = glob(["data.tar.*"]),
```

Comment:
technically no, but rules_oci only supports `.gz` at the moment.

Reply:
`tar` does autodetection, so I figured it would be OK to be liberal in what you accept and let `tar` sort it out. It'll return non-zero if it can't find `zstd` to do the decompression, so in theory it'd behave the same way for `.zip` or something more exciting.

Reply:
that's true.